workflow/kfdrc-somatic-variant-workflow.cwl

```diff
@@ -948,16 +877,15 @@ steps:
           $(self.secondaryFiles.filter(function(e) { return e.basename.search(/.dict$/) != -1 })[0])
       calling_regions: coding_sequence_regions
       supplement_vcfs:
-        source: [strelka2/strelka2_protected_outputs, mutect2/mutect2_protected_outputs]
+        source: [strelka2/strelka2_protected_outputs, mutect2/mutect2_protected_outputs, lancet_input_vcf]
         valueFrom: |
           $(self.filter(function(e) { return e != null }).map(function(e) { return e.filter(function(i) { return i.basename.search(/(.vcf|.vcf.gz)$/) != -1 })[0] }))
```

Comment:
So yeah the `valueFrom` won't play nice with `lancet_input_vcf`. Specifically the `[0]` at the end, you'll only ever end up taking the first VCF from `lancet_input_vcf`.